metadata-ingestion/src/datahub/ingestion/source/hex/hex.py

```diff
@@ -93,9 +101,27 @@ class HexSourceConfig(
         default=True,
         description="Set ownership identity from owner/creator email",
     )
+    include_lineage: bool = Field(
```

Comment:
@hsheth2 should we add some mention here to the dependency on backend version?

Reply:
we should mention (1) the limitations of the Hex lineage and (2) that it requires snowflake/bigquery/redshift with queries v2 enabled + a server with your model changes

We should probably put those notes in hex.md, and then just refer to them here

Reply:
addressed in commit a50887d34be492264bc50788647419f5e19749b4